engine/samples/cars/main.cpp

```diff
@@ -17,30 +17,47 @@ using cubos::core::gl::Grid;
 using cubos::core::gl::Palette;
 using namespace cubos::engine;
 
+/// [Get handles to assets]
 static const Asset<Grid> CarAsset = AnyAsset("059c16e7-a439-44c7-9bdc-6e069dba0c75");
 static const Asset<Palette> PaletteAsset = AnyAsset("1aa5e234-28cb-4386-99b4-39386b0fc215");
+/// [Get handles to assets]
 
+/// [Spawner resource]
 struct Spawner
 {
     float timer = 0.0F;
     int x = -1;
     int y = -1;
 };
+/// [Spawner resource]
 
-static void settings(Write<Settings> settings)
+/// [Set settings]
+static void settingsSystem(Write<Settings> settings)
 {
     settings->setString("assets.io.path", SAMPLE_ASSETS_FOLDER);
 }
+/// [Set settings]
```

Comment:
I think I would omit this from the docs - its not related to the actual sample, and its something we must do on all samples using assets, which shouldn't be needed in a 'real world' scenario.